notNeededPackages.json

```
@@ -3738,6 +3738,10 @@
             "libraryName": "rgb-hex",
             "asOfVersion": "3.0.0"
         },
+        "riderize__passport-strava-oauth2": {
+            "libraryName": "@riderize/passport-strava-oauth2",
+            "asOfVersion": "1.1.1"
```

Added on commit https://github.com/auth0-community/auth0-socketio-jwt/commit/72f3846fcf8b9c5e8d3d0850806464152292a02d which belongs to v1.0.9
Changelog for v1.0.9: https://github.com/Riderize/passport-strava-oauth2/releases/tag/1.0.9
```suggestion
            "asOfVersion": "1.0.9"
```

---

This is not possible, there is already a `@types/riderize__passport-strava-oauth2@1.1.0` published on npm. Thus, I simply took the next version.

---

This is supposed to be the number of the version that added type declarations for this package, it's not the version of the `@types` package

---

The checks won't pass. You can try it out yourself with `npm run test-all`. This is actually supposed to be the version of the `@types` package that will be published with the obsoleted warning.